Mimir/GraphQL/TypeExtensions/SheetQueryTypeExtension.cs

```diff
@@ -0,0 +1,33 @@
+using Lib9c.GraphQL.Objects;
+using Lib9c.GraphQL.Types;
+using Mimir.Enums;
+using Mimir.GraphQL.Queries;
+using Mimir.Repositories;
+
+namespace Mimir.GraphQL.TypeExtensions;
+
+public class SheetQueryTypeExtension : ObjectTypeExtension<Query>
+{
+    protected override void Configure(IObjectTypeDescriptor<Query> descriptor)
+    {
+        descriptor
+            .Field("sheet")
+            .Argument("planetName", a => a.Type<NonNullType<PlanetNameType>>())
+            .Argument("sheetName", a => a.Type<NonNullType<SheetNameType>>())
+            .Type<NonNullType<SheetType>>()
+            .ResolveWith<SheetQueryTypeExtension>(t => GetSheet(default!, default!, default!));
+    }
+
+    private static async Task<SheetObject> GetSheet(
+        string planetName,
+        string sheetName,
+        [Service] TableSheetsRepository tableSheetsRepository)
+    {
+        var csv = await tableSheetsRepository.GetSheet(planetName, sheetName, SheetFormat.Csv);
+        return new SheetObject
+        {
+            Name = sheetName,
+            Csv = csv,
```

Comment:
how about doing Base64 encoding?

Reply:
Can you explain more?

Reply:
> Currently, the value returned by the `{network}/tablesheet/{name}` path is not base64 encoded, and encoding it would change the value that Mimir users expect, so I think this should be a separate issue.

Reply:
Since no one is using GQL yet, it should be fine.

Reply:
OK I got it.

Reply:
Applied!